subsys/bluetooth/audio/vocs_internal.h

```diff
@@ -71,6 +71,10 @@ struct bt_vocs_server {
 	struct bt_vocs_cb *cb;
 
 	struct bt_gatt_service *service_p;
+
+	struct k_work_delayable state_notify_work;
+	struct k_work_delayable location_notify_work;
+	struct k_work_delayable output_desc_notify_work;
```

Comment:
Could we settle with a single k_work for these, and using the same bitfield behavior that you implemented for MCS?

Reply:
Sure, the delay was dependent on the number of retries but I changed it after @fredrikdanebjer suggestion.
I'll update the PR